low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -177,7 +175,7 @@ class SSPLBundleGenerate(object):
 
     def get_consul_data(self):
         self.consul_folder_name = f"consul_sspl_data-{str(int(time.time()))}"
-        consuldump = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/bin/consuldump.py -n {self.consul_folder_name} -l /tmp/{sspl_bundle_tmp}/ -d"
+        consuldump = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/low-level/files/opt/seagate/sspl/bin/consuldump.py -n {self.consul_folder_name} -l /tmp/{sspl_bundle_tmp}/ -d"
```

Comment:
path to be defined & used from a constant, for reusability

Reply:
Done